web/new_alt_text_manager.js

```diff
@@ -208,19 +197,34 @@ class NewAltTextManager {
   }
 
   #toggleAI(value) {
+    if (!this.#uiManager || this.#hasAI === value) {
+      return;
+    }
+    this.#hasAI = value;
     this.#dialog.classList.toggle("noAi", !value);
-    this.#toggleTitle();
+    this.#toggleTitleAndDisclaimer();
   }
 
-  #toggleDisclaimer(value = null) {
-    if (!this.#uiManager) {
+  #toggleTitleAndDisclaimer() {
+    // Disclaimer is visible when the AI is loading or the user didn't change
+    // the guessed alt text.
+    const visible =
+      this.#isAILoading ||
+      (this.#guessedAltText && this.#guessedAltText === this.#textarea.value);
+    this.#disclaimer.classList.toggle("noDisclaimer", !visible);
```

Comment:
Previously this would be hidden using the "standard" `hidden`-rule, any reason for this now using a custom class?